batchflow/models/tf/mobilenet.py

```diff
@@ -219,8 +222,203 @@ def block(cls, inputs, filters, residual=False, strides=1, expansion_factor=6, w
+
+        x = conv_block(inputs, 'cna', num_filters, kernel_size=1, name='%s-exp' % name, strides=1, **kwargs)
+        x = depthwise_conv(x, kernel_size=3, strides=strides, padding='same',
+                           data_format=kwargs['data_format'], name='%s-depthwise' % name)
+        x = conv_block(x, 'nacn', filters, kernel_size=1, strides=1, name='%s-down' % name, **kwargs)
+
         if residual:
             x = inputs + x
         return x
+
+
+_V3_LARGE_DEFAULT_BODY = [
+    dict(repeats=1, filters=16, expansion_factor=1, strides=1, kernel_size=3, se_block=False),
+    dict(repeats=1, filters=24, expansion_factor=4, strides=2, kernel_size=3, se_block=False),
+    dict(repeats=1, filters=24, expansion_factor=3, strides=1, kernel_size=3, se_block=False,
+         residual=True),
+    dict(repeats=3, filters=40, expansion_factor=3, strides=2, kernel_size=5, se_block=True),
+    dict(repeats=1, filters=80, expansion_factor=6, strides=2, kernel_size=3, activation=h_swish, se_block=False),
+    dict(repeats=1, filters=80, expansion_factor=2.5, strides=1, kernel_size=3, activation=h_swish, se_block=False,
+         residual=True),
+    dict(repeats=2, filters=80, expansion_factor=2.3, strides=1, kernel_size=3, activation=h_swish, se_block=False,
+         residual=True),
+    dict(repeats=2, filters=112, expansion_factor=6, strides=1, kernel_size=3, activation=h_swish, se_block=True),
+    dict(repeats=3, filters=160, expansion_factor=6, strides=2, kernel_size=5, activation=h_swish, se_block=True),
+]
+
+class MobileNet_v3(TFModel):
+    """ MobileNet version 3
+
+    **Configuration**
+
+    inputs : dict
+        dict with 'images' and 'labels' (see :meth:`.TFModel._make_inputs`)
+
+    initial_block : dict
+        parameters for the initial block (default is 'cna', 32, 3, strides=2)
+
+    body : dict
+        layout : list of dict
+            a sequence of block parameters:
+            repeats : int
+            filters : int
+            expansion_factor : int
+            strides : int
+            kernel_size : int
+            h_swish : bool
+            se_block : bool
+            residual : bool
+
+        width_factor : float
+            multiplier for the number of channels (default=1)
+    """
+    @classmethod
+    def default_config(cls):
+        config = TFModel.default_config()
+        config['initial_block'].update(dict(layout='cna', filters=16, kernel_size=3, strides=2, activation=h_swish))
+        config['body'].update(dict(width_factor=1, layout=deepcopy(_V3_LARGE_DEFAULT_BODY)))
+        config['head'].update(dict(layout='cnavcacV', filters=[960, 1280, 2], pool_size=7, kernel_size=1,
+                                   activation=h_swish))
+        config['loss'] = 'ce'
+
+        return config
+
+    def build_config(self, names=None):
+        config = super().build_config(names)
+        config['head']['filters'][-1] = self.num_classes('targets')
```

Comment:
```suggestion
        config['head/filters'][-1] = self.num_classes('targets')
```

Reply:
This will fail if head has different layout (e.g. 'cnaV')

Reply:
why? this line changes defined in the default config list of filters. No conflicts If a layout has only one convolution - its num_filters will be changed.

Reply:
`head/filters` can be just `int`, not list

Reply:
fixed